apps/frontend/components/PlainTemplate.js

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+
+export default props => {
+  const {
+    schema,
+    id,
+    classNames,
+    label,
+    help,
+    required,
+    description,
+    errors,
+    children
+  } = props;
+
+  return (
+    <div data-testid="plain-template" className={classNames}>
```

Comment:
@duranmla let's keep https://www.npmjs.com/package/babel-plugin-jsx-remove-data-test-id in mind when we start deploying to production